### PR TITLE
Fixed a bug in the logging of cni isolator.

### PR DESCRIPTION
**Pull request by haoyixin**
the old errno would always be 0, and we can't get the reason that why execute ifconfig failed.

so, we use the return value of spawn for WSTRINGIFYing error in log.

and this fix format issue @haosdent

### CODE REVIEW

File: src/slave/containerizer/mesos/isolators/network/cni/cni.cpp [hunk 1]
```diff
@@ -1994,11 +1995,13 @@ int NetworkCniIsolatorSetup::execute()
...
-      cerr << "Failed to bring up the loopback interface in the new "
-           << "network namespace of pid " << flags.pid.get()
-           << ": " << os::strerror(errno) << endl;
-      return EXIT_FAILURE;
+    int spawn = os::spawn("ifconfig", {"ifconfig", "lo", "up"});
+
+    if (spawn != 0) {
+        cerr << "Failed to bring up the loopback interface in the new "
+             << "network namespace of pid " << flags.pid.get()
+             << ": " << (spawn < 0 ? "failed to fork process" : WSTRINGIFY(spawn)) << endl;
```
Comment (haosdent) [inline src/slave/containerizer/mesos/isolators/network/cni/cni.cpp]: Replace `failed` to `Failed`.

**Comment by qianzhangxa** [comment]
This PR can be closed in favour of https://reviews.apache.org/r/65306/.

**Comment by Gilbert88** [comment]
@haoyixin Could you close this PR in favor of https://reviews.apache.org/r/65306/

**Comment by greggomann** [comment]
Closing since the related RR has landed.

**Pull request closed without merging**